machine/delta/kernel/ismt.patch

```diff
@@ -0,0 +1,70 @@
+--- a/drivers/i2c/busses/i2c-ismt.c	2016-08-14 16:07:51.465407328 +0800
```

Comment:
Why is this patch necessary?  I have used ismt with kernel 4.1 on several machines without problems.

Reply:
We did the stress test in eeprom on  our platform and we found that there were read/write errors  hppened smetimes. So we added the patch file on our platform according  to our test result.